atlasdb-config/src/main/java/com/palantir/atlasdb/factory/ServiceDiscoveringAtlasSupplier.java

```diff
@@ -45,26 +46,31 @@
     private static final Logger log = LoggerFactory.getLogger(ServiceDiscoveringAtlasSupplier.class);
     private static final ServiceLoader<AtlasDbFactory> loader = ServiceLoader.load(AtlasDbFactory.class);
 
+    private static final double GOLDEN_RATIO = (Math.sqrt(5) + 1.0) / 2.0;
+    private static final int QUICK_MAX_BACKOFF_SECONDS = 1000;
```

Comment:
Do we feel like there are scenarios where frequently polling will actually overload the KVS if it's trying to start up?